packages/PriceLockup/PriceLockup.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './PriceLockup.modules.scss'
+import joinClassNames from '../../shared/utils/joinClassNames'
+import safeRest from '../../shared/utils/safeRest'
+
+/**
+ * @version ./package.json
+ */
+
+const priceStyle = (element, size) => {
+  switch (size) {
+    case 'medium':
+      return styles[`${element}--medium`]
+    case 'large':
+      return styles[`${element}--large`]
+    default:
+      return ''
+  }
+}
+
+const PriceLockup = ({
+  size,
+  price,
+  priceBold,
+  topLabel,
+  signDirection,
+  rateLabel,
+  bottomLabel,
+  ...rest
+}) => {
+  const priceLabelClass = joinClassNames(styles.priceLabel, priceStyle('priceLabel', size))
+
+  const priceBottomLabelClass = joinClassNames(
+    styles.priceBottomLabel,
+    priceStyle('priceBottomLabel', size)
+  )
+
+  const priceSignClass = joinClassNames(styles.priceSign, priceStyle('priceSign', size))
+
+  const priceValueClass = joinClassNames(
+    styles.priceValue,
+    priceStyle('priceValue', size),
+    priceBold ? styles['priceValue--bold'] : ''
+  )
+
+  const priceRateClass = joinClassNames(styles.priceRate, priceStyle('priceRate', size))
+
+  const PriceValueSign = () => {
+    if (signDirection === 'right') {
+      return (
+        <span>
+          <span className={priceValueClass}>{price}</span>
+          <span className={priceSignClass}>&#36;</span>
+        </span>
+      )
+    }
+    return (
+      <span>
+        <span className={priceSignClass}>&#36;</span>
```

Comment:
For accessibility reasons, please change this to a `<p>` tag. (Although if you're using the FlexGrid change, this will become unnecessary)